Rubberduck.Core/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -22,6 +24,25 @@
 
 namespace Rubberduck.UI.Inspections
 {
+    [Flags]
+    public enum InspectionResultsFilter
+    {
+        None = 0,
+        Hint = 1,
```

Comment:
the pattern would dictate this to be `1 << 1` which is obviously nonsense, because that's what `Suggestion` should be. There's a gap here.

Reply:
That always throws me (I generally use `2 ^ 0`, `2 ^ 1`, `2 ^ 2`, ...) but I used what seems to be RD convention and face-planted it...  Will fix.